spec/lib/facilities/local_store_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'facilities/local_store'
+
+RSpec.describe Facilities::LocalStore do
+  let(:url) { 'https://www.example.com/VBA_Facilities' }
+  let(:bulk_client_stub) { instance_double('Facilities::BulkClient') }
+  let(:adapter_stub) { class_double('VBAFacilityAdapter') }
+  subject { described_class.new(url, nil, adapter_stub) }
+
+  before(:each) do
+    subject.instance_variable_set(:@client, bulk_client_stub)
```

Comment:
Will look into a better way of doing this, agree it feels hinky - `@client` (and `@conn` in the other spec) are not exposed as attributes so I was puzzling over how to stub them.